erdpy/config.py

```diff
@@ -5,3 +5,7 @@
 ROOT_FOLDER_NAME = "ElrondSCTools"
 DEFAULT_GASPRICE = 200000000000
 DEFAULT_GASLIMIT = 50000000
+
+MIN_GAS_LIMIT = 50000
+GAS_PER_DATA_BYTE = 1500
+SYSTEM_SC_COST = 5000000
```

Comment:
Perhaps create a map (dict) that resembles this economics config? So that, when they change, we simple adjust the values here? Can be constants as well, not necessarily a dictionary. Say, `SystemSC_Stake_Cost` etc. 

```

[MetaChainSystemSCsCost]
    Stake               = 5000000
    UnStake             = 5000000
    UnBond              = 5000000
    Claim               = 5000000
    Get                 = 5000000
    ChangeRewardAddress = 5000000
    ChangeValidatorKeys = 5000000
    UnJail              = 5000000
    ...
```

Reply:
refactored.